.gitignore

```diff
@@ -1,2 +1,3 @@
 pydocstringformatter.egg-info
 dist/
+__pycache__/
```

Comment:
Isn't `pycache` automatically ignored like `.mypy_cache` etc? It's ignored for me.

Reply:
I can add this to my global gitignore :)